src/QsCompiler/CompilationManager/DataStructures.cs

```diff
@@ -29,10 +29,10 @@ internal class CodeLine
         /// lesser are considered "non-string" states.
         ///
         /// Here is an example of how these states relate to Q# text:
-        ///     let str = <NoOpenString> " <OpenString> Hello World" <NoOpenString>;
-        ///     let str2 = <NoOpenString> $" <OpenInterpolatedString> Hello
-        ///         { <OpenInterpolatedArgument> " <OpenStringInOpenInterpolatedArgument> World" <OpenInterpolatedArgument> }
-        ///         <OpenInterpolatedString> " <NoOpenString>;
+        ///     let str = [NoOpenString] " [OpenString] Hello World" [NoOpenString];
```

Comment:
@ScottCarda-MS, any preference on this change?

`<` and `>` cannot be used in XML comments, but they can be encoded as `&lt;` and `&gt;`, respectively. I opted for square brackets for in-source readability.

Reply:
Works fine for me. Thanks!